src/components/Onboarding/Screens/Apps/ConvictionVotingSettings/index.js

```diff
@@ -0,0 +1,323 @@
+                  proposal.
+                </Help>
+              </Fragment>
+            }
+            minValue={1}
+            value={minThreshold}
+            onChange={handleMinThresholdChange}
+          />
+          <Button
+            size="mini"
+            css={`
+              align-self: flex-end;
+            `}
+            label="Advanced Settings"
+            onClick={() => setOpenSettingsModal(true)}
+          />
+          <AdvancedSettingsModal
+            stakeOnProposalPct={stakeOnProposal}
+            visible={openSettingsModal}
+            onClose={handleCloseModal}
+          />
+          <div
+            css={`
+              display: flex;
+              flex-direction: column;
+              margin-top: ${4 * GU}px;
+            `}
+          >
+            <div
+              css={`
+                color: ${theme.contentSecondary};
+                margin-bottom: ${3 * GU}px;
+                ${textStyle('body3')};
+              `}
+            >
+              The following are not actual parameters. You can adjust them to
+              play around with the charts.
+            </div>
+            <PercentageField
+              label={
+                <Fragment>
+                  Requested Amount
+                  <Help hint="What is Requested Amount?">
+                    <strong>Requested Amount</strong> is the percentage of funds
+                    being requested by the proposal displayed in the charts.
+                  </Help>
+                </Fragment>
+              }
+              value={requestedAmount}
+              onChange={handleRequestedAmountChange}
+            />
+            <PercentageField
+              label={
+                <Fragment>
+                  Stake On Proposal
+                  <Help hint="What is Stake On Proposal?">
+                    <strong>Stake On Proposal</strong> is the percentage of your
+                    total stake tokens allocated to the proposal being displayed
+                    in the charts.
+                  </Help>
+                </Fragment>
+              }
+              value={stakeOnProposal}
+              onChange={handleStakeOnProposalChange}
+            />
+            <PercentageField
+              label={
+                <Fragment>
+                  Stake On Other Proposals
+                  <Help hint="What is Stake On Other Proposals?">
+                    <strong>Stake On Other Proposals</strong> is the percentage
+                    of your total stake tokens allocated to proposals other than
+                    the one being displayed in the charts.
+                  </Help>
+                </Fragment>
+              }
+              value={stakeOnOtherProposals}
+              onChange={handleStakeOnOtherProposalsChange}
+            />
+          </div>
+        </div>
+        <div
+          css={`
+            margin-left: ${5 * GU}px;
```

Comment:
```suggestion
            margin-left: ${10 * GU}px;
```

If we make this margin bigger it would stick to the width of other screens and help with clearness.